packages/react-components/src/util/pingback.ts

```diff
@@ -9,7 +9,7 @@ const firePingback = (actionType: PingbackActionType) => (
     attributes: PingbackAttributes = {}
 ) =>
     pingback({
-        gif,
+        analyticsResponsePayload: gif.analytics_response_payload,
```

Comment:
@dleitee sometimes gifs don't have an `analytics_response_payload`, even though I typed it as always there (maybe we should change that?). 

Regardless, here I think we should not fire this if `gif.analytics_response_payload` doesn't exist. 

```typescript 
const firePingback = (actionType: PingbackActionType) => (
    gif: IGif,
    userId: string | number | undefined,
    target: HTMLElement,
    attributes: PingbackAttributes = {}
) =>
    gif.analytics_response_payload && pingback({
        analyticsResponsePayload: gif.analytics_response_payload,
        userId,
        actionType,
        attributes: { position: JSON.stringify(getClientRect(target)), ...attributes },
    })

Reply:
ok, not a problem ;D